src/Engine/MapController.java

```diff
@@ -393,7 +394,8 @@ public void onStateChange()
           myGame.gameMap.getLocation(targ).setHighlight(true);
         }
         // Create an option selector to keep track of where we are.
-        myGame.setCursorLocation(myGameInputHandler.getCoordinateOptions().get(myGameInputOptionSelector.getSelectionNormalized()));
+        myGame
+            .setCursorLocation(myGameInputHandler.getCoordinateOptions().get(myGameInputOptionSelector.getSelectionNormalized()));
```

Comment:
This is a real wart.

Reply:
Yes

Reply:
I'm all for changing the code style... but I don't think my habit of compulsively formatting is gonna go away.